### ISSUE TITLE
XY Precision setting

### ISSUE THREAD

**Issue by MadTooler** [sameer/svg2gcode#34]
While I have had very good and smooth running results with different controllers running the svg2gcode output, I find the 13 or so decimal places of mm a bit excessive.  It would be helpful for output precision settings for a resulting smaller file size, and in cases like mine, a helpful preformatting where I am taking the gcode and handling it a bit before I drip feed it over to a controller.

Line numbers are also sometimes useful.

Thank you for sharing your project and your time.

**Comment by sameer** [comment]
> I find the 13 or so decimal places of mm a bit excessive.

Agreed, right now I think the formatting is using the raw floating point values. The main problem is that rounding/truncating could cause the g-code to violate the tolerance

> Line numbers are also sometimes useful.

Will create a separate issue for this
